packages/resolver/test/globalnpm.ts

```diff
@@ -5,6 +5,7 @@ import * as sinon from "sinon";
 import path from "path";
 import fs from "fs";
 import { describe, it } from "mocha";
+import { EOL} from "os";
```

Comment:
Style note since Husky's not running at the moment, `{ EOL}` should be `{ EOL }` :P

Reply:
Updated. Thanks.